Makefile

```diff
@@ -71,65 +80,67 @@ orion-server-images:
 	docker pull orionbcdb/orion-server:$(ORION_VERSION)
 	docker image tag orionbcdb/orion-server:$(ORION_VERSION) orionbcdb/orion-server:latest
 
+INTEGRATION_TARGETS = download-fabric integration-tests-iou
+INTEGRATION_TARGETS += integration-tests-atsacc 
+INTEGRATION_TARGETS += integration-tests-chaincode-events
+INTEGRATION_TARGETS += integration-tests-atsafsc
+INTEGRATION_TARGETS += integration-tests-twonets
+INTEGRATION_TARGETS += integration-tests-pingpong
+INTEGRATION_TARGETS += integration-tests-stoprestart
+
 .PHONY: integration-tests
-integration-tests:
-	cd ./integration/fabric/iou; ginkgo $(GINKGO_TEST_OPTS) .
-	cd ./integration/fabric/atsa/chaincode; ginkgo $(GINKGO_TEST_OPTS) .
-	cd ./integration/fabric/events/chaincode; ginkgo $(GINKGO_TEST_OPTS) .
-	cd ./integration/fabric/atsa/fsc; ginkgo $(GINKGO_TEST_OPTS) .
-	cd ./integration/fabric/twonets; ginkgo $(GINKGO_TEST_OPTS) .
-	cd ./integration/fsc/pingpong/; ginkgo $(GINKGO_TEST_OPTS) .
-	cd ./integration/fsc/stoprestart; ginkgo $(GINKGO_TEST_OPTS) .
+integration-tests: $(INTEGRATION_TARGETS)  
 
 .PHONY: integration-tests-iou
-integration-tests-iou:
-	cd ./integration/fabric/iou; ginkgo $(GINKGO_TEST_OPTS) .
+integration-tests-iou: download-fabric
+	cd ./integration/fabric/iou; export FAB_BINS=$(FAB_BINS); ginkgo $(GINKGO_TEST_OPTS) .
 
 .PHONY: integration-tests-iou-hsm
-integration-tests-iou-hsm:
+integration-tests-iou-hsm: download-fabric
 	@echo "Setup SoftHSM"
 	@./ci/scripts/setup_softhsm.sh
 	@echo "Start Integration Test"
-	cd ./integration/fabric/iouhsm; ginkgo $(GINKGO_TEST_OPTS) .
+	cd ./integration/fabric/iouhsm; export FAB_BINS=$(FAB_BINS); ginkgo $(GINKGO_TEST_OPTS) .
 
 .PHONY: integration-tests-iouorionbe
-integration-tests-iouorionbe: orion-server-images
-	cd ./integration/fabric/iouorionbe; ginkgo $(GINKGO_TEST_OPTS) .
+integration-tests-iouorionbe: orion-server-images download-fabric
+	cd ./integration/fabric/iouorionbe; export FAB_BINS=$(FAB_BINS); ginkgo $(GINKGO_TEST_OPTS) .
 
 .PHONY: integration-tests-atsacc
-integration-tests-atsacc:
-	cd ./integration/fabric/atsa/chaincode; ginkgo $(GINKGO_TEST_OPTS) .
+integration-tests-atsacc: download-fabric
```

Comment:
do we still need this dependency target here?
The CI downloads the binary before starting the tests and in the README you already tell that the binaries must be downloaded. No?

Reply:
Thanks @adecaro looks like I missed one when I removed downloading fabric from the targets per previous comment, I will remove it.